Adding link for guide-docker to "what will you learn" section + updating Copyright (#140)
OpenLiberty/guide-containerize: README.adoc

== What you'll learn

> You can easily deploy your microservices in different environments in a lightweight and portable manner by using containers. From development to production and across your DevOps environments, you can deploy your microservices consistently and efficiently with containers. You can run a container from a container image. Each container image is a package of what you need to run your microservice or application, from the code to its dependencies and configuration. If you're new to the development of applications in containers, you might want to start with the https://openliberty.io/guides/docker.html[Using Docker containers to develop microservices^] guide first.

Reviewer: ```suggestion
From development to production, and across your DevOps environments, you can deploy your microservices in a lightweight and portable manner by using containers.
You can run a container from a container image. Each container image is a package of what you need to run your microservice or application, from the code to its dependencies and configuration. If you're new to the development of applications in containers, you might want to start with the https://openliberty.io/guides/docker.html[Using Docker containers to develop microservices^] guide before you work through this guide.
```
